lib/google_contacts_api/groups.rb

```diff
@@ -23,5 +23,28 @@ def get_groups(params = {})
       end
       GoogleContactsApi::GroupSet.new(response.body, @api)
     end
+
+    def create_group(title)
```

Comment:
Also, eventually we might want to support more than just setting the title, but it's a good first step.